src/gp/src/GPMSAOptions.C

```diff
@@ -550,16 +564,18 @@ GPMSAOptions::set_final_scaling
 
       V minScenario(*m_simulationScenarios[0]);
 
+      // Only use simulation data for scaling.
       for (unsigned int i=1; i < m_simulationScenarios.size(); ++i)
         min_max_update(minScenario, maxScenario,
                        (*m_simulationScenarios[i]));
 
-      // To match Matlab code, only use simulation data for scaling.
-/*
+      // But check the experimental data, and yell at the user if it
+      // falls outside the simulation data range, because they
+      // probably don't want to be extrapolating with their GP.
       for (unsigned int i=0; i < m_experimentScenarios.size(); ++i)
         min_max_update(minScenario, maxScenario,
-                       (*m_experimentScenarios[i]));
-*/
+                       (*m_experimentScenarios[i]),
+                       /*warn_on_update =*/ "scenario parameter");
```

Comment:
If `min_max_update` is still being called on the experiment scenarios, then isn't the min and max including the experiment scenarios in the normalization?